tests/test-cases/Group6-VIC-Machine/6-03-Delete.robot

```diff
@@ -203,14 +236,15 @@ Delete VCH moved from its RP
 
 
 Delete VCH moved to root RP and original RP deleted
```

Comment:
this test is just not appropriate for CI at all, need to be in nightly

Reply:
What is inappropriate about it?